src/helpers/xml-builder.js

```diff
@@ -1,2390 +1,2473 @@
+
+          tableRowFragment.import(rowSpanCellFragment);
+        }
+      }
+      const tableCellFragment = await buildTableCell(
+        column,
+        { ...modifiedAttributes, maximumWidth: maximumColumnWidth },
+        rowSpanMap,
+        columnIndex,
+        docxDocumentInstance
+      );
+      columnIndex.index++;
+
+      tableRowFragment.import(tableCellFragment);
+    }
+  }
+
+  if (columnIndex.index < rowSpanMap.size) {
+    const rowSpanCellFragments = buildRowSpanCell(rowSpanMap, columnIndex, modifiedAttributes);
+    if (Array.isArray(rowSpanCellFragments)) {
+      for (let iteratorIndex = 0; iteratorIndex < rowSpanCellFragments.length; iteratorIndex++) {
+        const rowSpanCellFragment = rowSpanCellFragments[iteratorIndex];
+
+        tableRowFragment.import(rowSpanCellFragment);
+      }
+    }
+  }
+
+  tableRowFragment.up();
+
+  return tableRowFragment;
+};
+
+const buildTableGridCol = (gridWidth) =>
+  fragment({ namespaceAlias: { w: namespaces.w } })
+    .ele('@w', 'gridCol')
+    .att('@w', 'w', String(gridWidth));
+
+const buildTableGrid = (vNode, attributes) => {
+  const tableGridFragment = fragment({ namespaceAlias: { w: namespaces.w } }).ele('@w', 'tblGrid');
+  if (vNodeHasChildren(vNode)) {
+    const gridColumns = vNode.children.filter((childVNode) => childVNode.tagName === 'col');
+    const gridWidth = attributes.maximumWidth / gridColumns.length;
+
+    for (let index = 0; index < gridColumns.length; index++) {
+      const tableGridColFragment = buildTableGridCol(gridWidth);
+      tableGridFragment.import(tableGridColFragment);
+    }
+  }
+  tableGridFragment.up();
+
+  return tableGridFragment;
+};
+
+const buildTableGridFromTableRow = (vNode, attributes) => {
+  const tableGridFragment = fragment({ namespaceAlias: { w: namespaces.w } }).ele('@w', 'tblGrid');
+  if (vNodeHasChildren(vNode)) {
+    const numberOfGridColumns = vNode.children.reduce((accumulator, childVNode) => {
+      const colSpan =
+        childVNode.properties.colSpan ||
+        (childVNode.properties.style && childVNode.properties.style['column-span']);
+
+      return accumulator + (colSpan ? parseInt(colSpan) : 1);
+    }, 0);
+    const gridWidth = attributes.maximumWidth / numberOfGridColumns;
+
+    for (let index = 0; index < numberOfGridColumns; index++) {
+      const tableGridColFragment = buildTableGridCol(gridWidth);
+      tableGridFragment.import(tableGridColFragment);
+    }
+  }
+  tableGridFragment.up();
+
+  return tableGridFragment;
+};
+
+const buildTableBorders = (tableBorder) => {
+  const tableBordersFragment = fragment({ namespaceAlias: { w: namespaces.w } }).ele(
+    '@w',
+    'tblBorders'
+  );
+
+  const { strokes, colors, ...borders } = tableBorder;
+
+  Object.keys(borders).forEach((border) => {
+    if (borders[border]) {
+      const borderFragment = buildBorder(
+        border,
+        borders[border],
+        0,
+        colors[border] || '000000',
```

Comment:
Can we make the defaults a constant referenced from the defaults? I see it in multiple spots. Also add this change to the options and readme. For example `0`, `000000`, `nil`